extensions/health/src/types.ts

```diff
@@ -11,6 +11,9 @@ export type HealthOptions = {
   healthPath: string;
   readyPath: string;
   livePath: string;
+  openApiSpec?: {
+    disabled?: boolean;
+  };
```

Comment:
Do you see possibility to have other options for `openApiSpec`? If not, let's use `openApiSpec?: boolean`.